src/docfx/build/xref/InternalXrefSpec.cs

```diff
@@ -15,7 +15,9 @@ internal record InternalXrefSpec(SourceInfo<string> Uid, string Href, FilePath D
 
     public string? SchemaType { get; init; }
 
-    public Dictionary<string, Lazy<JToken>> XrefProperties { get; } = new Dictionary<string, Lazy<JToken>>();
+    public bool IsNameLocalizable { get; set; }
+
+    public Dictionary<string, Lazy<JToken>> XrefProperties { get; } = new();
```

Comment:
Should `IsLocalizable` be combined with the specific Xref property?

Reply:
We can consider title only up to now.